src/types.ts

```diff
@@ -202,18 +236,22 @@ export interface Game {
     pass?: boolean;
     setActivePlayers?: boolean;
   };
-  endIf?: (G: any, ctx: Ctx) => any;
-  onEnd?: (G: any, ctx: Ctx) => any;
-  playerView?: (G: any, ctx: Ctx, playerID: PlayerID) => any;
-  plugins?: Array<Plugin>;
+  endIf?: (G: G, ctx: Ctx | CtxWithPlugins) => any;
+  onEnd?: (G: G, ctx: Ctx | CtxWithPlugins) => any;
+  playerView?: (G: G, ctx: Ctx | CtxWithPlugins, playerID: PlayerID) => any;
+  plugins?: Array<Plugin<any, any, G>>;
   processMove?: (
-    state: State,
+    state: State<G, Ctx | CtxWithPlugins>,
```

Comment:
I think you can leave this one as it is. (In any case, `processMove` is an internal field that is not intended for public consumption.)